doc/src/manual/documentation.md

```diff
@@ -81,7 +81,15 @@ As in the example above, we recommend following some simple conventions when wri
    ...
    """
    ```
-5. Include any code examples in an `# Examples` section.
+5. Provide hints to similar functions
```

Comment:
Better use "related" here too. `exp` and `log` are definitely related, but I wouldn't consider them as "similar". Also missing ending period.